build.sbt

```diff
@@ -19,6 +19,7 @@ lazy val root = (project in file("."))
     retrieveManaged := true,
     initialCommands in console := "import uk.gov.hmrc._",
     parallelExecution in Test := false,
+    testOptions in Test := Seq.empty,
```

Comment:
Might be worth a comment explaining why this is necessary, as I suspect we'll quickly forget ...

Reply:
I have added a comment now. Thanks.